### ISSUE TITLE
Reason 11 rack plugin stops working after a few minutes

### ISSUE THREAD

**Issue by WarlockStones** [robbert-vdh/yabridge#155]
### Thanks for giving yabridge a shot!

- [X] I read through both the troubleshooting and the known issues sections, and my issue wasn't listed there

### Problem description

After a few minutes of using Reason 11 as a plugin, the plugin window freezes and can not be interacted with.

### What did you expect to happen?

I expect to continue be able to use Reason 11 Rack Plugin in my DAW.

### What actually happened?

The plugin installs fine without any issues. It runs and works as expected for a few minutes. Then the window freezes and becomes non intractable. The plugin is running in the background and I can play sounds with my MIDI keyboard. However, no further changes to the plugin window can be made. Removing and adding the effect again does not work, it is still frozen.  By restarting the DAW (Reaper) the plugin works for a few minutes before breaking again.

When the plugin is running it is spamming these errors:
```
[Wine STDERR] 0104:fixme:dwrite:fontface_get_design_advance Transform is not supported.
[Wine STDERR] 0104:fixme:d2d:d2d_geometry_sink_AddArc iface 00000000207C0578, arc 000000000011D2C0 stub!
[Wine STDERR] 0104:fixme:d2d:d2d_device_context_DrawGeometry Ignoring stroke style 000000001DC32630.
```

Then when it crashes it throws me some .NET errors.
```
[Wine STDERR] 0680:err:module:import_dll Library mscoree.dll (which is needed by L"C:\\windows\\Microsoft.NET\\Framework\\v4.0.30319\\mscorsvw.exe") not found
[Wine STDERR] 0680:err:module:LdrInitializeThunk Importing dlls for L"C:\\windows\\Microsoft.NET\\Framework\\v4.0.30319\\mscorsvw.exe" failed, status c0000135
[Wine STDERR] 00dc:err:service:process_send_command service protocol error - failed to write pipe!
[Wine STDERR] 00dc:fixme:service:delayed_autostart_callback Delayed auto-start service L"clr_optimization_v4.0.30319_32" failed to start: 1053
[Wine STDERR] 0688:err:module:import_dll Library mscoree.dll (which is needed by L"C:\\windows\\Microsoft.NET\\Framework64\\v4.0.30319\\mscorsvw.exe") not found
[Wine STDERR] 0688:err:module:LdrInitializeThunk Importing dlls for L"C:\\windows\\Microsoft.NET\\Framework64\\v4.0.30319\\mscorsvw.exe" failed, status c0000135
[Wine STDERR] 00dc:err:service:process_send_command service protocol error - failed to write pipe!
[Wine STDERR] 00dc:fixme:service:delayed_autostart_callback Delayed auto-start service L"clr_optimization_v4.0.30319_64" failed to start: 1053
```

The errors are the same every try.

### Operating system

Fedora 35

### How did you install yabridge?

Directly from the releases page

### yabridge version

3.7.0

### yabridgectl version

3.7.0

### Wine version

wine-7.0-rc2

### Plugin

Reason 11 Rack Plugin

### Plugin type

VST3

### Plugin architecture

64-bit

### Host

REAPER v6.43

### Desktop environment or WM

Awesome 4.3 on Xorg

### GPU model

NVIDIA 1070

### GPU drivers and kernel

NVIDIA Proprietary 495.46 on Linux 5.15.11-200.fc35.x86_64

### Debug log

[yabridge.log](https://github.com/robbert-vdh/yabridge/files/7809647/yabridge.log)

### Anything else?

I am using PipeWire 0.3.40 that comes with Fedora 35.
I have the Reason 11 installed using Bottles.

I Installed reason 11 using a guide i found on Reddit.
[https://www.reddit.com/r/reasoners/comments/qy1dsh/using_the_reason_rack_plugin_on_linux/](url)

The guide mentions that "in the preferences tab of bottles, uncheck "use dxvk" and "use vkd3d" otherwise everything will start glitching out after a few minutes"
I have these thing unchecked. But in a log file i get information about Vulkan. Is it running Vulkan and causing these issues? 
[yabridge-host.exe.so_d3d11.log](https://github.com/robbert-vdh/yabridge/files/7809660/yabridge-host.exe.so_d3d11.log)

**Comment by robbert-vdh** [comment]
I can't really help you with Reason Rack since I've never used it myself, but other people in the yabridge Discord seem to not having any issues with it. You're using DXVK, so if people say that Reason Rack doesn't play well with DXVK for whatever reason, then you should probably try it with stock WineD3D instead. And unrelated to this, but you're also using [rtkit in PipeWire](https://github.com/robbert-vdh/yabridge#troubleshooting-common-issues) which will definitely cause issues later on (hence why yabridge is showing you notifications about it).

**Comment by WarlockStones** [comment]
I reinstalled Reason 11 using PlayOnLinux to get rid of dxvk.

This gave me a new error when it freezes.
```
[Wine STDERR] err:   D3D11: Failed to create GDI DC
[Wine STDERR] 0104:err:gdi:alloc_gdi_handle out of GDI object handles, expect a crash
```

I found a mention that installing GDI Plus could help.
Installing gdiplus removed those errors from the log but it still freezes after a fem minutes. 
However, It did fix the issue with having to restart the whole DAW. I can now simply restart the FX window in REAPER. It is now usable!

So to conclude. Yes, dxvk does screw with Reason 11.
Installing gdiplus helped.
It is usable but some issues are causing the window to freeze.
These freezes are only effecting the window and its interactivity. Otherwise it works well.

Thanks! I posted my new logs if interested.
[yabridge.log](https://github.com/robbert-vdh/yabridge/files/7826670/yabridge.log)
